packages/askui-nodejs/src/lib/copy-example-project.ts

```diff
@@ -1,24 +1,53 @@
-import { Command } from 'commander';
+import { Command, OptionValues } from 'commander';
 import path from 'path';
 import fs from 'fs-extra';
 import { getPathToNodeModulesRoot } from '../utils/path';
+import { logger } from './logger';
 
 const createProgram = () => {
   const program = new Command('askui');
   program.usage('<command> [options]');
   return program;
 };
-function copyExampleProject() {
+
+async function replaceStringInFile(filePath: string, replace: string, replacement: string) {
+  try {
+    const data = await fs.readFile(filePath, 'utf8');
+    const result = data.replace(replace, replacement);
+    await fs.writeFile(filePath, result, 'utf8');
+  } catch (error: unknown) {
+    logger.error(`Could not replace '${replace}' with ${replacement} in file ${path}`);
```

Comment:
`${replacement}` should also be surrounded by quotation marks

Reply:
And for the _path_ also! I was sloppy there...

Reply:
@adi-wan-askui Changed and pushed!